lrauv_description/test/worlds/flat_world.sdf

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" ?>
+<!--
+  Development of this module has been funded by the Monterey Bay Aquarium
+  Research Institute (MBARI) and the David and Lucile Packard Foundation
+-->
+<sdf version="1.6">
+  <world name="buoyant_tethys">
```

Comment:
Should change the world name to match the file unless there's a reason to keep `buoyant_tethys`, e.g. used in some code - I don't see any.